src/layout/page-layout.scss

```diff
@@ -0,0 +1,384 @@
+            margin-right: calc(var(--Layout-column-gap) * -1);
+          }
+
+          .PageLayout-content {
+            // stylelint-disable-next-line primer/spacing
+            margin-left: var(--Layout-column-gap);
+          }
+        }
+      }
+
+      &.PageLayout--panePos-end {
+        .PageLayout-pane {
+          // stylelint-disable-next-line primer/borders
+          border-left: $border-width solid $Layout-divider-color;
+        }
+
+        &:not(.PageLayout--columnGap-none) {
+          .PageLayout-pane {
+            // stylelint-disable-next-line primer/spacing
+            padding-left: calc(var(--Layout-column-gap) - #{$border-width});
+            // stylelint-disable-next-line primer/spacing
+            margin-left: calc(var(--Layout-column-gap) * -1);
+          }
+
+          .PageLayout-content {
+            // stylelint-disable-next-line primer/spacing
+            margin-right: var(--Layout-column-gap);
+          }
+        }
+      }
+    }
+
+    // sticky pane
+
+    &.PageLayout--isPaneSticky {
+      .PageLayout-pane {
+        position: sticky;
+        top: 0;
+        max-height: 100vh;
+        overflow: auto;
+      }
+    }
+
+    // content width
+
+    [class^='PageLayout-content-centered-'] {
+      max-width: calc(var(--Layout-content-width) + var(--Layout-pane-width) + var(--Layout-column-gap));
+      margin-right: auto;
+      margin-left: auto;
+    }
+
+    &.PageLayout--hasPaneDivider {
+      [class^='PageLayout-content-centered-'] {
+        max-width: calc(var(--Layout-content-width) + var(--Layout-pane-width) + (var(--Layout-column-gap) * 2));
+      }
+    }
+
+    &.PageLayout--panePos-start {
+      [class^='PageLayout-content-centered-'] > [class^='container-'] {
+        margin-left: 0;
+      }
+    }
+
+    &.PageLayout--panePos-end {
+      [class^='PageLayout-content-centered-'] > [class^='container-'] {
+        margin-right: 0;
+      }
+    }
+
+    @each $breakpoint in map-keys($breakpoints) {
+      .PageLayout-content-centered-#{$breakpoint} {
+        --Layout-content-width: #{map-get($breakpoints, $breakpoint)};
+      }
+    }
+
+    // pane width
+
+    @each $breakpoint in map-keys($sidebar-width) {
+      @include breakpoint($breakpoint) {
+        --Layout-pane-width: #{map-get($sidebar-width, $breakpoint)};
+      }
+    }
+
+    &.PageLayout--paneWidth-narrow {
+      @each $breakpoint in map-keys($sidebar-narrow-width) {
+        @include breakpoint($breakpoint) {
+          --Layout-pane-width: #{map-get($sidebar-narrow-width, $breakpoint)};
+        }
+      }
+    }
+
+    &.PageLayout--paneWidth-wide {
+      @each $breakpoint in map-keys($sidebar-wide-width) {
+        @include breakpoint($breakpoint) {
+          --Layout-pane-width: #{map-get($sidebar-wide-width, $breakpoint)};
+        }
+      }
+    }
+  }
+
+  // responsive variants
+
+  @media (max-width: #{map-get($breakpoints, $Layout-responsive-variant-max-breakpoint) - 0.02px}) {
```

Comment:
I think we should treat small viewports as the default, not large viewports.

Here's where that opinion comes from:
- We treat mobile as the default in the other responsive utilities in Primer CSS
- It encourages designers and engineers to design and write markup for their content with the assumption that the user is viewing in a wide viewport, and then we have to try and squish it down to fit in a mobile viewport.
- If the content is marked up for mobile, the more critical information will likely come earlier in the source order. This could give a better experience to people using assistive technology like screenreaders.

Are there benefits to treating the wide viewport as the default? I haven't been thinking about this component as much as you, so I could be misunderstanding something.

Reply:
> I think we should treat small viewports as the default, not large viewports.

@mperrotti that's basically what's happening, with the difference that the small viewport styling gets limited to a `max-width` media query. Otherwise the larger viewport would have to do a lot of work to reset the narrow variants. Maybe you got that feeling because of the order of the selectors?

> Are there benefits to treating the wide viewport as the default? I haven't been thinking about this component as much as you, so I could be misunderstanding something.

The wide viewport styling is not the default, it's behind a `breakpoint` mixin that only gets applied to `md` breakpoints and wider:

```scss
 @include breakpoint($Layout-responsive-variant-max-breakpoint) { }
```

Reply:
I'll mark this as resolved for now, let me know if there are any questions!